tock/hours/views.py

```diff
@@ -504,6 +505,9 @@ def get_context_data(self, **kwargs):
             'timecard_notes': TimecardNote.objects.enabled(),
             'unsubmitted': not self.object.submitted,
             'reporting_period': reporting_period,
+            'excluded_from_billability': json.dumps(list(Project.objects.excluded_from_billability().values_list('id', flat=True))),
```

Comment:
Really minor, but I don't think you should need to convert the values_list to list. Given flat=True, I think it should be a list already.

Reply:
`values_list` does not return a `list`, it returns a `ValuesListQuerySet`.  `flat=True`  only means to return results as single values rather than one-tuples.
https://docs.djangoproject.com/en/3.0/ref/models/querysets/#values-list

Reply:
I still think `json.dumps(ProjectProject.objects.excluded_from_billability().values_list('id', flat=True))` should be fine, but it's so minor it's not even worth the time I've spent typing this. My point was that when using `flat=True` it's list-like enough to not need to cast to `list` but really, it's not important.